Hedgehog-Markets/hedgehog-escrow
# Clean up a program code and tests

## Programs

- Clean up instructions and documentation.
- Fix confusing error names.
- Remove redundant `rent` sysvar account from `InitializeMarket` instruction.
- Move `authority` and `fee_wallet` addresses from account info to parameters for `InitializeGlobalState` and `SetGlobalState`.

## Tests

- Add test and build scripts.
- Add linter for tests (eslint).
- Add prettier to check and standardize formatting.
- Reduce duplicate code in tests.

## Misc

- Upgrade anchor ts patch to https://github.com/Hedgehog-Markets/anchor/commit/0adb822b7a0e569a7a9ea48d894b0d353770f1e5.
- Upgrade to `anchor-lang` version `0.25.0`.
